community/ignis/RB_overview.ipynb

```diff
@@ -287,35 +291,178 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7ff057deb550>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(8, 6))\n",
+    "ax = plt.subplot(1, 1, 1)\n",
+    "\n",
+    "# Plot the essence by calling plot_rb_data\n",
+    "rb_fit.plot_rb_data(0, ax=ax, add_label=True, show_plt=False)\n",
+    "    \n",
+    "# Add title and label\n",
+    "ax.set_title('%d Qubit RB'%(nQ), fontsize=18)\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Note:* In the graph above $p$ is called \"alpha\"."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The intuition behind RB\n",
+    "\n",
+    "The depolarizing channel has parameter $p$, and works like this: with probability $p$, the state remains the same as before; with probability $1-p$, the state becomes the totally mixed state.\n",
```

Comment:
If you look at our papers we use $\alpha$ for $p$ here. Also you have $1-p^m$ should be $(1-p)^m$. The last equation didn't render.